developersociety/django-template
# Add initial homepage migration

By default the initial homepage you see is the Wagtail "it works!" page - which is a standard wagtail page which we always have to replace. It would be easier if the site had the HomePage model being the home page.

Wagtail does something similar to this in their project template: https://github.com/wagtail/wagtail/tree/v2.7.3/wagtail/project_template/home/migrations - which I've copied and tweaked slightly.

To test:

```
mktmpenv
cookiecutter --no-input --checkout homepage gh:developersociety/django-template wagtail=y
cd projectname
nvm use
make npm-install pip-install-local
dropdb --if-exists projectname_django
createdb projectname_django
./manage.py migrate
make django-dev-createsuperuser
./manage.py runserver
```

Then go to http://127.0.0.1:8000/ - you'll see our standard homepage.
